modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenOperation.java

```diff
@@ -117,7 +117,7 @@ public boolean isRestfulIndex() {
      * @return true if act as Restful show method, false otherwise
      */
     public boolean isRestfulShow() {
-        return "GET".equals(httpMethod) && isMemberPath();
```

Comment:
@wing328 I'm sorry, it was my fault. I changed those methods when I started implementing the generator and right after I noticed they were not necessary, but I forgot to revert the changes :). Now it should be ok. 

FYI, I added `equalsIgnoreCase` rather than `equals` because the `httpMethod` variable is not always uppercase. Do you agree?